server/views/pages/reportDetailMacro.njk

```diff
@@ -28,23 +37,23 @@
             {{ fieldValue('Created by', data.reporterName, 'reporter-name')  }}
           </p>
           <p class="govuk-!-margin-bottom-1">
-            {{ fieldValue('Date and time', data.submittedDate | formatDate('D MMMM YYYY, HH:mm'), 'submitted-date')  }}
+            {{ fieldValue('Date and time', data.submittedDate | formatDate('D MMMM YYYY, HH:mm'), 'submitted-date') }}
           </p>
           <p class="govuk-!-margin-bottom-1">
-            {{ fieldValue('Prisoner name', data.offenderDetail.displayName, 'prisoner-name')  }}
+            {{ fieldValue('Prisoner name', data.offenderDetail.displayName, 'prisoner-name') }}
           </p>
           <p class="govuk-!-margin-bottom-1">
-            {{ fieldValue('Prisoner number', data.offenderDetail.offenderNo, 'prisoner-number')  }}
+            {{ fieldValue('Prisoner number', data.offenderDetail.offenderNo, 'prisoner-number') }}
           </p>
           <p class="govuk-!-margin-bottom-1">
-            {{ fieldValue('Incident number', data.incidentId, 'incident-number')  }}
+            {{ fieldValue('Incident number', data.incidentId, 'incident-number') }}
           </p>
       </div>    
     </div>
   </div>
 {% endmacro %}
 
-{% macro detail(data, bookingId, incidentId, user, prison) %}
+{% macro detail(data, bookingId, incidentId, user, print) %}
```

Comment:
this is used in 3 files but I can only see it updated in one. 

If the prison param was not being used was it also definitely not being specified in the other 2 places?